clj-refactor.el

```diff
@@ -2689,9 +2689,9 @@ str/split => split"
   (let ((name (cljr--normalize-symbol-name symbol)))
     (cond
      ((string-match-p "\\w+\\.\\w+" name)
-      (thread-last name (s-split "\\.") last car cljr--symbol-suffix))
+      (thread-last name (split-string "\\.") last car cljr--symbol-suffix))
```

Comment:
we decided to go down the behavioural testing route not the unit testing one. which is essence was a good choice i think. so i suppose just leave it ;)